tests/typecheck/test_views.yml

```diff
@@ -19,3 +19,14 @@
         queryset = MyModel.objects.all()
 
         def perform_destroy(self, instance: MyModel) -> None: ...
+
+- case: test_build_api_request
+  main: |
+    from django.http.request import HttpRequest
+    from rest_framework.views import APIView
+
+    request = HttpRequest()
```

Comment:
It is better to write: `request: HttpRequest`

Reply:
Done